css/properties/text-emphasis-position.json

```diff
@@ -47,6 +47,39 @@
             "deprecated": false
           }
         },
+        "auto": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "132"
+              },
+              "firefox_android": "mirror",
```

Comment:
I'm curious why https://github.com/mdn/browser-compat-data/pull/24658 didn't include this. Looks like there is no [`text-emphasis-position` test](https://mdn-bcd-collector.gooborg.com/tests/css/properties/text-emphasis-position) for this value yet.

@queengooborg @Elchi3 What needs to happen or be changed for the collector to pick up and test this value? Does `@webref/css` just need to start including this value [here](https://github.com/w3c/webref/blob/01a31d3d9e7fcb898b17bd30442e8bf7c20ef6cd/tr/css/css-text-decor.json#L271-L300)?

Reply:
This doesn't seem to be in the spec, see https://drafts.csswg.org/css-text-decor-3/#text-emphasis-position-property. Therefore it can't be extracted into webref of course. https://github.com/w3c/csswg-drafts/issues/1198 says "Resolved" in the title because I guess there is consensus on this but maybe no one has written spec text for it yet.

Reply:
Thanks for clarifying, Florian, that makes sense.